src/classes/SRVC_Contact_PrimaryAffiliations.cls

```diff
@@ -90,9 +90,14 @@ public virtual with sharing class SRVC_Contact_PrimaryAffiliations {
         try {
             for (MODL_PrimaryAfflMapper eachModel: afflMappingsModelsList) {
                 if (contact.get(eachModel.getContactFieldApiName()) == null) {
+                    if (eachModel.getIsValid() == false){
+                        throw new ERR_ExceptionHandler.AffAccountRecordTypeMappingsException(Label.afflAccoutMappingError);
+                    }
                     continue;
                 }
```

Comment:
There seems to be a small bug that was revealed in testing. The exception should be thrown when it's not null.
```suggestion
                if (contact.get(eachModel.getContactFieldApiName()) == null) {
                    continue;
                }

                if (eachModel.getIsValid() == false){
                    throw new ERR_ExceptionHandler.AffAccountRecordTypeMappingsException(Label.afflAccoutMappingError);
                }
```